src/Altinn.App.Core/Internal/Pdf/PdfService.cs

```diff
@@ -225,4 +229,34 @@ private static string GetValidFileName(string fileName)
         fileName = Uri.EscapeDataString(fileName.AsFileName(false));
         return fileName;
     }
+
+    private static string GetFooterContent(Instance instance)
+    {
+        TimeZoneInfo norwegianTimeZone = TimeZoneInfo.FindSystemTimeZoneById("Europe/Oslo");
+        DateTimeOffset norwegianNow = TimeZoneInfo.ConvertTime(DateTimeOffset.UtcNow, norwegianTimeZone);
+
+        string dateGenerated = norwegianNow.ToString("G", new CultureInfo("nb-NO"));
+
+        string altinnReferenceId = instance.Id.Split("/")[1].Split("-")[4];
+
+        string footerTemplate =
+            $@"<div style='font-family: Inter; font-size: 12px; width: 100%; display: flex; flex-direction: column; gap: 12px; padding: 0 70px 0 70px;'>
+                <span class='title'></span>
```

Comment:
Since you are adding the page title in the PDF footer, maybe we should have unit/cypress tests in app-frontend verifying that this always shows the `appName - serviceOwner` in PDF mode? This could be changed when we start working on navigation 🤔

Reply:
I dont think the title here is related to pages. It is just the document title, which seems to get created based on the filename. The filename gets created using the `"appName"` and `"serviceOwner"` text resources. If the text resources does not exist, the title will just be the "raw" appname (the repo name i think?).

Reply:
I am pretty sure it is the page title, as in the `<title>` element. Chrome knows nothing about file-names as it simply returns the PDF as a base64 string, and then we determine the filename afterwards

Reply:
You are correct, it seems to be the `title` tag in the apps `index.cshtml` file:
<img width="410" alt="image" src="https://github.com/user-attachments/assets/b8d7a46a-d7e3-4001-a4bc-82f06107619f">

Reply:
That is because we control it in app-frontend 😄 which is why I think we should add tests for it there. Because if I remember correctly, one wish was that the title reflects the current page you are on in the form, and we could inadvertantly break the PDF title if we don't handle that case specifically